nrfutil/sandboxTypes.ts

```diff
@@ -17,13 +17,11 @@ export type FeatureClassification =
     | 'nrf-external-confidential'
     | 'nrf-external';
 
-export type Task = {
+export type Task<T = void> = {
```

Comment:
Why have you chosen `void` as the default type of `T`?

Typescript Documentation clearly state that void is only used to describe that a function returns nothing: https://www.typescriptlang.org/docs/handbook/2/functions.html#void.

I would think that this should either be `undefined` or `null`, and probably `null`, as `undefined` may give the assumption that it could at any point become defined? 🤔